react/features/welcome/components/WelcomePage.web.js

```diff
@@ -108,9 +108,11 @@ class WelcomePage extends AbstractWelcomePage {
                     className = { `welcome ${showAdditionalContent
                         ? 'with-content' : 'without-content'}` }
                     id = 'new_welcome_page'>
+                    <div className = 'welcome-watermark'>
```

Comment:
Not introduced by this PR, but do we keep the old welcome page somewhere? Why do we call it new_welcome_page? :)

Reply:
I wonder...if this were introduced while I was converting over to the new welcome page design and never dropped the new prefix. It sounds like something I would do and forget to clean up.